docs/details.md

```diff
@@ -203,6 +203,8 @@ Affects how quick the services reading Ethereum events realize there's a new blo
 
 * **`block_submit_every_nth`** - how many new Ethereum blocks must be mined, since previous submission **attempt**, before another block is going to be formed and submitted.
 
+* **`block_submit_when_n_txs`** - pending transaction count that will trigger block submission faster than in required Ethereum's blocks (see: `block_submi_every_nth`). When not set defaults to infinity, effecively switching this feature off.
```

Comment:
1. typo, `block_submi_every_nth`
2. how about we name it `block_submit_after_n_txs`?
3. I think it could be explained more clearly: `maximum number of transactions in a block. If block reaches this number of transactions it is submitted to plasma contracts` or something similar.